validator/client/beacon-api/propose_attestation_test.go

```diff
@@ -258,86 +248,41 @@ func TestProposeAttestationElectra(t *testing.T) {
 		},
 		{
 			name:                 "nil attestation",
-			expectedErrorMessage: "attestation can't be nil",
+			expectedErrorMessage: "attestation is nil",
 		},
 		{
 			name: "nil attestation data",
-			attestation: &ethpb.AttestationElectra{
-				AggregationBits: testhelpers.FillByteSlice(4, 74),
-				Signature:       testhelpers.FillByteSlice(96, 82),
-				CommitteeBits:   testhelpers.FillByteSlice(8, 83),
+			attestation: &ethpb.SingleAttestation{
+				AttesterIndex: 74,
+				Signature:     testhelpers.FillByteSlice(96, 82),
+				CommitteeId:   83,
 			},
-			expectedErrorMessage: "attestation can't be nil",
+			expectedErrorMessage: "attestation is nil",
 		},
 		{
 			name: "nil source checkpoint",
-			attestation: &ethpb.AttestationElectra{
-				AggregationBits: testhelpers.FillByteSlice(4, 74),
+			attestation: &ethpb.SingleAttestation{
+				AttesterIndex: 74,
 				Data: &ethpb.AttestationData{
 					Target: &ethpb.Checkpoint{},
 				},
-				Signature:     testhelpers.FillByteSlice(96, 82),
-				CommitteeBits: testhelpers.FillByteSlice(8, 83),
+				Signature:   testhelpers.FillByteSlice(96, 82),
+				CommitteeId: 83,
 			},
 			expectedErrorMessage: "attestation's source can't be nil",
 		},
 		{
 			name: "nil target checkpoint",
-			attestation: &ethpb.AttestationElectra{
-				AggregationBits: testhelpers.FillByteSlice(4, 74),
+			attestation: &ethpb.SingleAttestation{
+				AttesterIndex: 74,
 				Data: &ethpb.AttestationData{
 					Source: &ethpb.Checkpoint{},
 				},
-				Signature:     testhelpers.FillByteSlice(96, 82),
-				CommitteeBits: testhelpers.FillByteSlice(8, 83),
+				Signature:   testhelpers.FillByteSlice(96, 82),
+				CommitteeId: 83,
 			},
 			expectedErrorMessage: "attestation's target can't be nil",
 		},
-		{
-			name: "nil aggregation bits",
-			attestation: &ethpb.AttestationElectra{
-				Data: &ethpb.AttestationData{
-					Source: &ethpb.Checkpoint{},
-					Target: &ethpb.Checkpoint{},
-				},
-				Signature:     testhelpers.FillByteSlice(96, 82),
-				CommitteeBits: testhelpers.FillByteSlice(8, 83),
-			},
-			expectedErrorMessage: "attestation's bitfield can't be nil",
-		},
-		{
-			name: "nil signature",
-			attestation: &ethpb.AttestationElectra{
-				AggregationBits: testhelpers.FillByteSlice(4, 74),
-				Data: &ethpb.AttestationData{
-					Source: &ethpb.Checkpoint{},
-					Target: &ethpb.Checkpoint{},
-				},
-				CommitteeBits: testhelpers.FillByteSlice(8, 83),
-			},
-			expectedErrorMessage: "attestation signature can't be nil",
```

Comment:
I removed it because we had two functions in the codebase checking if an attestation is nil. One is `helpers.ValidateNilAttestation` and the other one is `validateNilAttestation` on the VC side. The only difference between them was signature validation which is missing in the `helpers` package. I decided it's better to just have a single implementation and I expect the one in `helpers` to be more accurate. I am not exactly sure what happens though when the BN receives an attestation without a signature if the function in `helpers` doesn't fail.